projects/packages/search/src/widgets/class-search-widget.php

```diff
@@ -131,28 +117,35 @@ public function activate_search() {
 	public function widget_admin_setup() {
 		wp_enqueue_style(
 			'widget-jetpack-search-filters',
-			plugins_url( 'search/css/search-widget-admin-ui.css', __FILE__ ),
+			plugins_url( 'css/search-widget-admin-ui.css', __FILE__ ),
 			array(),
-			JETPACK__VERSION
+			// TODO: We'll need our own version number.
```

Comment:
Could you explain what you mean here? Are you referring to making `Helper::get_asset_version` independent of the Jetpack plugin?

Reply:
Yep that's right - I figure I could just change it to use the `Asset` class, and `Helper::get_asset_version` is to be removed in #22419.

Reply:
Excellent!